doc/man1/openssl-version.pod.in

```diff
@@ -77,6 +78,11 @@ The random number generator source settings.
 
 The OpenSSL CPU settings info.
 
+=item B<-w>
+
+The OpenSSL WININSTALLCONTEXT build time variable, if set.
+Used for computing Windows registry key names
+
```

Comment:
Deserve a HISTORY section, no ?

Reply:
not sure I follow.  Are you suggesting a HISTORY section that describes how we used build time defined variables for paths on windows previously, and why this was introduced?

Reply:
Made an attempt at a history section